docs/suspensive.org/src/pages/docs/jotai/motivation.ko.mdx

```diff
@@ -0,0 +1,57 @@
+import { Steps } from 'nextra/components'
+
+# 어떤 이유로 사용하나요?
+
+@suspensive/jotai는 아래와 같은 동기로 사용됩니다.
+
+<Steps>
+
+### Jotai의 비동기 atom을 Suspense와 함께 더 직관적으로 사용할 수 있습니다.
+
+Jotai의 atom은 비동기 읽기/쓰기를 지원합니다. @suspensive/jotai의 `<Atom/>`과 함께라면 놀라울 정도로 쉽게 비동기 atom을 사용할 수 있습니다.
+
+Jotai의 비동기 atom은 비동기 흐름을 처리하기 위해 Suspense를 활용하고 있습니다. Atom이 비동기 동작을 시작하게 되면 부모의 Suspense에게 Promise의 pending 상태를 위임합니다.
+
+[@suspensive/react](https://suspensive.org/ko/docs/react/motivation)와 함께 사용하여 Suspense를 강력하고 편리하게 사용할 수 있습니다.
+
+### Async Atom를 선언적으로 사용할 수 있습니다.
+
+제공되는 컴포넌트는 Jotai와 동일한 인터페이스의 Props를 가지며 선언적으로 사용할 수 있습니다. 이는 [@suspensive/react-query의 SuspenseQuery 처럼](https://suspensive.org/ko/docs/react-query/SuspenseQuery), 같은 depth에서 Suspense를 발생시키는 것이 무엇인지 명확하게 표현할 수 있습니다.
+
+```tsx
+import { Atom } from '@suspensive/jotai'
+import { Suspense } from '@suspensive/react'
+import { UserProfile } from '~/components'
+import { userAtom } from "~/atoms";
+
+const Example = () => (
+  <Suspense fallback="loading...">
+    <Atom atom={userAtom}>{([data]) => <UserProfile key={data.id} {...data} />}
+  </Suspense>
+)
+```
+
+### Jotai의 커뮤니티 리소스와 함께 사용할 수 있습니다.
+
+Jotai에는 [Query](https://jotai.org/docs/extensions/query), [Cache](https://jotai.org/docs/extensions/cache) 등 다양한 커뮤니티 리소스가 있습니다. 커뮤니티 리소스에서 발생되는 Suspense 또한 지원됩니다.
```

Comment:
![Screenshot_20240715_092334_Chrome](https://github.com/user-attachments/assets/c3dcd74f-222a-49d3-b727-6ce869221978)

Is it really community resources? https://github.com/jotaijs/jotai-tanstack-query when I find github, It's not commnity resources. It's just extension. Double check correct word please

Reply:
Instead of referring to it as a "community resource" in Jotai, it might be better to call it "Extensions".

Reply:
> Instead of referring to it as a "community resource" in Jotai, it might be better to call it "Extensions".

I also think it's better to call them Extensions. I hope it is delivered without distortion. It would be better to use the words as they are exposed rather than making assumptions.